demo/example.md

```diff
@@ -0,0 +1,15 @@
+## Markdown Renderer
+
+Example:
+
+```html
+<paper-toolbar>
+  <paper-icon-button icon="menu"></paper-icon-button>
+  <div class="title">Title</div>
+  <paper-icon-button icon="more"></paper-icon-button>
+</paper-toolbar>
+```
+
+_Nifty_ features.
+
+[![Build status](https://travis-ci.org/PolymerElements/marked-element.svg?branch=master)](https://travis-ci.org/PolymerElements/marked-element)
```

Comment:
is this to test images? Use of the build status is confusing here, consider using a different image.